### ISSUE TITLE
add replication percentage

### ISSUE THREAD

**Issue by juliangruber** [datproject/dat-desktop#327]
This is something only we can offer, so it would be cool to show off with it! For a dat that you're sharing, it would be awesome to show how much of it is replicated in the entire network. So say the replication percentage is `50%`, that would mean that half of the dat's data is already available from other peers.

**Comment by juliangruber** [comment]
This gives us another dimension for a dat's health, and is just super handy in general.

**Comment by yoshuawuyts** [comment]
Yeah sounds cool, think this would tie into a "details" view (#260) - probably need that detail view for metadata editing anyway (grant requirement) so could be a cool bonus

**Comment by juliangruber** [comment]
What about instead this is another dimension on top of the progress bar? When you're sharing a dat that you already locally, the progress bar is kind of useless anyways:

<img width="192" alt="screen shot 2017-04-06 at 13 59 14" src="https://cloud.githubusercontent.com/assets/10247/24753146/3c2baa5a-1ad1-11e7-9ec7-8642fb05cea4.png">

So instead, we could use a different shade of green to render a progress bar "on top".

**Comment by yoshuawuyts** [comment]
@juliangruber interesting, sounds like you're right and  there's def options there - think @Kriesse could best answer how / what about this

**Comment by Kriesse** [comment]
I like it! We could show a darker green bar on top of your local 100% green bar.
But it's a concept that is not very intuitive and won't be clear just from the viusals or labels, so we'd need a tiny "?" icon next to it, that would explain it.

**Comment by juliangruber** [comment]
> so we'd need a tiny "?" icon next to it, that would explain it.

sgtm!

**Comment by juliangruber** [comment]
let's work on this once the edit in place feature landed :)

**Comment by Kriesse** [comment]
👍

**Comment by Kriesse** [comment]
## Mockup

![image](https://cloud.githubusercontent.com/assets/53684/24754580/5a5f3bb2-1ad7-11e7-9111-8380c73c75a9.png)


![image](https://cloud.githubusercontent.com/assets/53684/24754591/659e9040-1ad7-11e7-8547-f7e316808200.png)

**Comment by Kriesse** [comment]
@juliangruber does this make sense?
(The copy in the tooltip is awkward, need to improve that.)

**Comment by yoshuawuyts** [comment]
Copy idea: "20% of the data is currently not available on the network"
where s/20%/the actual value/ - that way people get more exact details, and
start relating the grayed out area with data not yet being available -
kinda useful for both beginners and power users

**Comment by juliangruber** [comment]
ah, yeah this is the case where you don't have everything yet, that makes sense too!

**Comment by karissa** [comment]
Did any users say they needed this?

**Comment by juliangruber** [comment]
not that i'm aware of, but also not that they were aware of this being possible.

i as a user would like this feature.

**Comment by karissa** [comment]
Can we prioritize the history and selective download and put this on the backburner? Unless at least 5 of our users asks for it (wed don't count as users :P)
